aip/0216.md

```diff
@@ -60,13 +60,57 @@ The field referencing the `State` enum in a resource **should** behave and be
 documented as "Output only", in accordance with [AIP-203](./0203.md).
 
 APIs **should not** allow a `State` enum to be directly updated through an
-"update" method, and **should** instead use a custom method.
+"update" method, and **should** instead use a custom state transition method.
 
 This is because update methods are generally not expected to have side effects,
 and also because updating state directly implies that it is possible to set the
 state to any available value, whereas states generally reflect a resource's
 progression through a lifecycle.
 
+### State transition methods
+
+State transition methods should look like this:
```

Comment:
We should say what they are first -- a bit of background. E.g., "State transition methods are a special type of custom method that are responsible for XYZ. The method definition should look like the following:"

Reply:
Done